Src/Application/Score/MaritalStatus/Commands/UpdateMaritalStatusScore/UpdateMaritalStatusScoreCommandHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Net;
+using Application.Common.Interfaces;
+using Domain.Entities;
+using MediatR;
+using Shared;
+
+namespace Application.Score.MaritalStatus.Commands.UpdateMaritalStatusScore;
+
+public class UpdateMaritalStatusScoreCommandHandler : IRequestHandler<UpdateMaritalStatusScoreCommand, OperationResult<string>>
+{
+    private readonly IRepository<MaritalStatusScore> _maritalStatusScoreRepository;
+
+    public UpdateMaritalStatusScoreCommandHandler(IRepository<MaritalStatusScore> maritalStatusScoreRepository)
+    {
+        _maritalStatusScoreRepository = maritalStatusScoreRepository;
+    }
+
+    public async Task<OperationResult<string>> Handle(UpdateMaritalStatusScoreCommand request, CancellationToken cancellationToken)
+    {
+        var maritalStatus = await _maritalStatusScoreRepository
+            .FirstOrDefaultAsync(x => x.Id == request.Id, cancellationToken);
+
+        if (maritalStatus is null)
+        {
+            return OperationResult<string>.Failed(Resource.RecordNotFound);
+        }
+
+        _maritalStatusScoreRepository.Update(maritalStatus);
```

Comment:
You just getting data from the database but you are not performing any update on it and just saving that object itself without any change
@FarzanehMG